src/lib/study-backrefs.js

```diff
@@ -217,6 +217,10 @@ const shortnameOfNonNormativeDocs = [
   "webrtc-nv-use-cases"
 ];
 
+const matchAnchor = (url, anchor) => link => {
+  return link === (url + "#" + anchor) || link === (url + "#" + encodeURIComponent(anchor));
+};
+
```

Comment:
Given that I already forgot why we're doing this in practice, could you clarify the logic on why we're not simply doing a strict match somewhere, e.g. in a comment just before the function gets declared?